waiter/integration/waiter/token_request_test.clj

```diff
@@ -267,7 +287,9 @@
                                        :owner (retrieve-username)
                                        :run-as-user "i-do-not-exist-but-will-not-be-checked"
                                        :token token}
-                    response (post-token waiter-url token-description :query-params {"update-mode" "admin"})]
+                    response (post-token waiter-url token-description
+                                         :headers {"if-unmodified-since" (str last-update-time)}
```

Comment:
The timestamp should conform to the spec.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Unmodified-Since

Reply:
There's lots of instances to fix.  I'm only annotating this one.

Reply:
Done.